chainerx_cc/cmake/FindBLAS.cmake

```diff
@@ -171,6 +171,21 @@ macro(Check_Fortran_Libraries LIBRARIES _prefix _name _flags _list _thread)
     mark_as_advanced(${_prefix}${_combined_name}_WORKS)
     set(_libraries_work ${${_prefix}${_combined_name}_WORKS})
   endif()
+  if(_libraries_work)
+    # NOTE: ChainerX uses CBLAS API. Search `libcblas.so` if CBLAS functions are
+    # not provided in the blas library found.
+    set(CMAKE_REQUIRED_LIBRARIES ${_flags} ${${LIBRARIES}} ${_thread})
+    check_function_exists("cblas_sgemm" ${_prefix}${_combined_name}_HAS_CBLAS)
+    if(NOT ${_prefix}${_combined_name}_HAS_CBLAS)
+        find_library(${_prefix}${_combined_name}_CBLAS_LIBRARY cblas)
+        if(${_prefix}${_combined_name}_CBLAS_LIBRARY)
+          set(${LIBRARIES} ${${LIBRARIES}} ${_prefix}${_combined_name}_CBLAS_LIBRARY)
```

Comment:
This line appends the variable name (e.g., BLAS_openblas_CBLAS_LIBRARY) to the `LIBRARIES` variable.
`set(${LIBRARIES} ${${LIBRARIES}} ${${_prefix}${_combined_name}_CBLAS_LIBRARY})` seems correct.